sqsworkers/crew.py

```diff
@@ -186,6 +197,71 @@ def poll_queue(self):
                         # continue with the next message and do not delete
                         pass
 
+    def bulk_poll_queue(self):
```

Comment:
This file is getting long, i would probably break it up somehow.  Subclass would be ok as Stephan mentioned, or another solution. But the refactor is getting much better!

Reply:
Thanks for helping me make it better. I would work on this offline and see if I can implement the subclass.